www-apps/gitea/gitea-1.7.4-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot systemd user
+
+EGO_PN="code.gitea.io/gitea"
+KEYWORDS="~amd64 ~arm"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="pam sqlite"
+
+CDEPEND="pam? ( sys-libs/pam )"
```

Comment:
You can improve it even better by getting rid of CDEPEND and make it `COMMON_DEPEND` instead. Any [x]DEPEND should be avoided from using, if they are added to later EAPIs.

Reply:
Thanks, will do.